src/tests/unit/tests/reports/components/report-sections/__snapshots__/report-body.test.tsx.snap

```diff
@@ -86,6 +86,7 @@ exports[`ReportBody renders 1`] = `
         "violations": Array [],
       }
     }
+    shouldAlertFailuresCount={false}
```

Comment:
Observation: Every instance in the snapshot sets shouldAlertFailuresCount to false. Should there be any test cases where it's set to true?

Reply:
should not as `ReportBody` is only used on the automated check report, which by definition, should not have any alert.